src/sentry/search/events/datasets/spans_indexed.py

```diff
@@ -237,6 +237,44 @@ def function_converter(self) -> Mapping[str, SnQLFunction]:
                     default_result_type="duration",
                     redundant_grouping=True,
                 ),
+                SnQLFunction(
+                    "trace_name",
+                    snql_aggregate=lambda args, alias: Function(
+                        "anyIf",
+                        [
+                            Column("segment_name"),
+                            Function(
+                                "or",
+                                [
+                                    Function("isNull", [Column("parent_span_id")]),
+                                    Function("equals", [Column("parent_span_id"), "00"]),
```

Comment:
Why are there both?

Reply:
Because it's a nullable column, but in the snuba consumer, they force a default of 0. so added both for further proofing